### PR TITLE
Rework modal

### PR DESCRIPTION
**Pull request by tgallice**

### CODE REVIEW

File: less/vendor/_nav-tabs.less [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  display: flex;
+  flex-direction: row;
+  background-color: @upf-gray-light;
+
+  li {
+    text-align: center;
+    flex-grow: 1;
+    float: none;
+
+    > a {
```
Comment (AlexisMontagne) [inline less/vendor/_nav-tabs.less]: `& > a` no?

Comment (tgallice) [reply less/vendor/_nav-tabs.less]: It's work :) http://less2css.org/#%7B%22less%22%3A%22.test%20%7B%5Cn%5Ct%3E%20.ok%20%7B%5Cn%20%20%20%20%5Ctcolor%3A%20%23fff%3B%5Cn%20%20%20%20%7D%5Cn%7D%22%7D

**Comment by AlexisMontagne** [comment]
Looks good to me, @phndiaye what do you think of it?

File: less/vendor/_modal.less [hunk 2]
```diff
@@ -4,26 +4,45 @@
...
-      background-color: @upf-primary-white;
-      border-bottom: 1px solid @upf-gray-dark;
+    .modal-title {
+      font-weight: bold;
+      color: @upf-primary-rock-blue;
       border: none;
+      padding-top: @spacing-xxx-sm;
+      padding-bottom: @spacing-md;
+
+      .text-size-5;
```
Comment (phndiaye) [inline less/vendor/_modal.less]: you can put the extended classes first, then the other rules

**Comment by phndiaye** [comment]
LGTM 👍

**Pull request merged**